test/commit-message-config.integration-test.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const shell = require('shelljs');
+const fs = require('fs');
+
+const mockers = require('./test-utils/mockers');
```

Comment:
In general, I have a preference against creating things named `utils` (also `common`, `shared` etc) - they kind of end up like the third drawer down in the kitchen, where everything that people didn't know where to put ends up. I think it's better to name the thing what it is, rather than utils.

Since we already have a `mocks` folder, do you think this file could be `test/mocks/jest-mocks` ?